# this message depicts a plan given by our planner

# msg header (seq, time_stamp, frame_id)
Header header

# current position (redundant)
geometry_msgs/Point32 cur_pos

# motion primitive
uint32 primitive
